### PR TITLE
OCM-11729 | ci: fix set pod level timeout env variable

### PR DESCRIPTION
**Pull request by aaraj7**

### CODE REVIEW

**Comment by aaraj7** [comment]
/pj-rehearse periodic-ci-openshift-rosa-master-e2e-rosa-day1-supplemental-f3

**Review by radtriste — COMMENTED** [review]
/lgtm

**Comment by aaraj7** [comment]
@yasun1 @radtriste The job still fails with the same error

File: ci-operator/step-registry/rosa/tests/rosa-tests-ref.yaml [hunk 1]
```diff
@@ -35,6 +35,9 @@ ref:
   - name: TEST_TIMEOUT
     default: "4h"
     documentation: Set time for the testing execution.
+  - name: TIMEOUT
+    default: "6h"
+    documentation: Set pod level timeout
```
Comment (yasun1) [inline ci-operator/step-registry/rosa/tests/rosa-tests-ref.yaml:40]: You made an incorrect update. This `timeout` is one of the pod attribute but not the global variable.
 E.g. https://github.com/openshift/release/blob/80c8d40788087fd7a6a7441f934e060d1587d56f/ci-operator/step-registry/rosa/cluster/wait-ready/cluster/rosa-cluster-wait-ready-cluster-ref.yaml#L10

**Comment by aaraj7** [comment]
/pj-rehearse periodic-ci-openshift-rosa-master-e2e-rosa-day1-supplemental-f3

**Pull request merged**